README.md

```diff
@@ -1 +1,79 @@
-# S-EL-DevSchool
\ No newline at end of file
+# Capstone Project: Web Dev Course 2022
+
+![Microverse bagde](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the first capstone project from microverse. this project is website for web developement school, were students can see new course programs and teachers .
+
+## Screenshots
+
+- **Mobile:**
+
+---
+![Mobile screenshot](./media/images/main_program.PNG)
+
+- **Laptops/Desktops:**
+
+---
+![Computer screenshot](./media/images/home_screenshot.PNG)
+
+## Built With
+
+- HTML/CSS
+- JS
+- Technologies:
+  - Git/Github
+  - Fontawesome
+
+## Live Demo
+
+[Live Demo Link](https://souad988.github.io/S-EL-DevSchool/)
+
+## Intro video
+
+> By watching this short video, you will have a better understanding of our project and mission.
+
+[S_EL_WEBDEV?]
```

Comment:
- [ ] **Required** There should be a video link in `Readme` or `PR` presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.